main/announcements/announcements.php

```diff
@@ -727,9 +728,12 @@
 
     echo '<input type="hidden" name="id" value="'.$announcement_to_modify.'" />';
 
-    $oFCKeditor = new FCKeditor('newContent') ;
-    $oFCKeditor->Width		= '100%';
-    $oFCKeditor->Height		= '300';
+
+    //$oFCKeditor = new FCKeditor('newContent') ;
```

Comment:
It seems something wrong with the spaces, it shoul be something like this:
<pre>
$oFCKeditor = new CKeditor();
$oFCKeditor->Width = '100%';
$oFCKeditor->Height = '300';
</pre>